packages/legend-query-builder/src/components/shared/BasicValueSpecificationEditor.tsx

```diff
@@ -35,6 +35,7 @@ import {
   DragPreviewLayer,
   CalculateIcon,
   InputWithInlineValidation,
+  type SelectComponent,
```

Comment:
if you don't mind, could we please clean imports in this file to move `type` above?

Reply:
Done!